app/styles/partials/utils.scss

```diff
@@ -236,4 +236,17 @@
 
 .mr-8 {
   margin-right: 2rem !important;
+  margin-bottom: -120px;
+}
+
+.event.time {
+  margin-left: 220px;
+}
+
+.event.name {
+  margin-left: 220px;
+}
+
+.event.location {
+  margin-left: 220px;
```

Comment:
these are classes from SemanticUI, do not add properties here

Reply:
ya hence i asked u that where would i get this styling

Reply:
> ya hence i asked u that where would i get the styling of these classes